src/submission/models.py

```diff
@@ -1045,6 +1045,13 @@ def production_assignment_or_none(self):
         except ObjectDoesNotExist:
             return None
 
+    @property
+    def citation_count(self):
+        article_link_count = self.articlelink_set.all().count()
```

Comment:
Should be `self.articlelink_set.count()` but I think the ORM is smart enough to not fetch the rows and just execute the count